fourcc.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import sys
+
+fourcc_include = sys.argv[1]
+
+ident_list = []
+with open(fourcc_include) as f:
+	for l in f.readlines():
+		parts = l.split()
```

Comment:
I'd still prefer something regex-based instead of ad-hoc parsing.
```python
fmt = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_code', flags=re.M)
mod = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_mod_code', flags=re.M)
```
`DRM_FORMAT_INVALID` would be a special case, as would some of the non-trivial modifiers.